### DISCUSSION TITLE
Use screenshots on po-level

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9440] [Q&A] [answered]
I like the feature to add screenshots.

But they are nto shown by default while translating. I have to associate them with the strings first. This is a lot of manual work.

Can I somehow ease this?

**Comment by nijel** [comment, marked as answer]
If you edit the screenshot, there is an “Automatically recognize” button to perform OCR on the image and make linking easier. This still requires manual clicking on the strings because matches are not always accurate.

Do you have other suggestions how to improve this?

**Reply by buhtz** [reply]
Great feature. But hidden. Not elegant currently.
But currently I don't see a better way to expose this feature to the users.
I'll think about it and open an Issue for it.
